src/routes/auth.js

```diff
@@ -69,4 +71,18 @@ router.post('/signup', signup);
  */
 router.post('/signin', signin);
 
+/**
+ * @swagger
+ *
+ * /auth/signout:
+ *   post:
+ *     description: Signs out a user
+ *     produces:
+ *       - application/json
+ *     responses:
+ *       200:
+ *         description: User successfully signed out
+ */
```

Comment:
Please, what parameters are you posting to this route and why is it not reflecting in the swagger comment documentation?

Reply:
I have fixed it. Thanks